lib/manageiq/release/release_tag.rb

```diff
@@ -13,6 +13,7 @@ def initialize(repo, branch:, tag:, dry_run: false)
       def run
         repo.fetch
         repo.checkout(branch)
+        repo.git.pull
```

Comment:
This is surprising because checkout does this already internally.

Reply:
I looked, but I don't think we actually do the 'pull'...